### ISSUE TITLE
Twitter account connected to kukai wallet got suspended

### ISSUE THREAD

**Issue by Lazyguy99** [kukai-wallet/kukai#244]
I got my twitter account which I used to make kukai wallet suspended and now I have no way of accessing my wallet.
Is there a way to retrieve my wallet?

**Comment by tezit** [comment]
Social sign in requires that the user maintain access to that social account. If you get banned by twitter you will lose your ability to sign in and access your wallet. In the future it might be possible to add additional recovery features for situations where a user gets banned, but that is not possible at this time.

**Comment by uloco** [comment]
how about changing your twitter handle. Will I be able to use my existing wallet if I change my twitter handle or is access lost?

**Comment by klassare** [comment]
> how about changing your twitter handle. Will I be able to use my existing wallet if I change my twitter handle or is access lost? 

Yes, you can change twitter handle without losing access to your wallet. Since your twitter id will remain the same.
